util/sorc/compile_gfs_util_jet.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+
+######################################################################
+#
+# Build executable GFS utility for GFS V16.0.0 
+#
+######################################################################
+
+LMOD_EXACT_MATCH=no
+source ../../sorc/machine-setup.sh > /dev/null 2>&1
+cwd=`pwd`
+
+if [ "$target" = "wcoss_dell_p3" ] || [ "$target" = "wcoss_cray" ] || [ "$target" = "jet" ] || [ "$target" = "hera" ] ; then
+   echo " "
+   echo " You are on WCOSS:  $target "
```

Comment:
You are not on WCOSS if you are on Hera or Jet

Reply:
Modified to display Hera and Jet